From: =?UTF-8?q?Rafa=C5=82=20Mi=C5=82ecki?= <rafal@milecki.pl>
Date: Thu, 9 Mar 2017 10:53:06 +0100
Subject: [PATCH] Revert "PCI: iproc: Request host bridge window resources"
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This reverts commit c3245a566400 ("PCI: iproc: Request host bridge
window resources"). It was passing local variable of
iproc_pcie_bcma_probe to the devm_request_pci_bus_resources. It meant
using unallocated memory for tracing resource which could easily result
in corruption and crashes.

Signed-off-by: Rafał Miłecki <rafal@milecki.pl>
---

--- a/drivers/pci/host/pcie-iproc.c
+++ b/drivers/pci/host/pcie-iproc.c
@@ -462,9 +462,6 @@ int iproc_pcie_setup(struct iproc_pcie *
 	struct pci_bus *bus;
 
 	dev = pcie->dev;
-	ret = devm_request_pci_bus_resources(dev, res);
-	if (ret)
-		return ret;
 
 	ret = phy_init(pcie->phy);
 	if (ret) {
